frontend/src/models/types.ts

```diff
@@ -189,7 +189,7 @@ export interface Schedule {
   yearMap: {
```

Comment:
On moving this file to the frontend--I believe this is a limitation of `create react app` since it configures the `webpack` to look for file only in the `frontend/src`. I think there's two options for us here. The first would be to `eject` which will then expose the `webpack.config.js` to us and we could then specify the root from where it should start looking for files. The second option would be to try something like this: `https://medium.com/@ryoldash/customize-webpack-config-of-react-app-created-with-create-react-app-7a78c7849edc ` using the `react-app-rewired` module to override `webpack.config.js`.

I'm not sure if this would work but we could also try restructuring the project as:
├───src/
│   ├───frontend
│   └───backend